src/Hydration.js

```diff
@@ -0,0 +1,38 @@
+class Hydration {
+  constructor(userID, hydrationData) {
+    this.userID = userID;
+    this.userData = hydrationData.filter(element => element.userID === userID);
+  }
+
+  calculateToalOunces() {
+    const total = this.userData.reduce(function(sum,current) {
```

Comment:
Just leaving a note here as a reminder we could refactor this callback to an arrow function for the sake of consistency